These dependencies are added in the Portfile. However, the py-pyqt5{-webengine} and py-pybojc
ports do not install an .egg-info file, so it appears that the package isn't present.
Removing it here from the setup.py file assures that it doesn't end up in the requires.txt
file and allows spyder to start. This is likely something that should be fixed in these ports,
These dependencies are added in the Portfile. However, the py-pyqt5{-webengine} and py-pybojc
ports do not install an .egg-info file, so it appears that the package isn't present.
Removing it here from the setup.py file assures that it doesn't end up in the requires.txt
file and allows spyder to start. This is likely something that should be fixed in these ports,
but for now just take the pragmatic approach.
--- setup.py.orig	2021-05-10 00:29:01.000000000 -0400
+++ setup.py	2021-05-11 22:36:42.000000000 -0400
@@ -201,7 +201,6 @@


 install_requires = [
-    'applaunchservices>=0.1.7;platform_system=="Darwin"',
     'atomicwrites>=1.2.0',
     'chardet>=2.0.0',
     'cloudpickle>=0.5.0',
@@ -224,8 +223,6 @@
     'pylint>=1.0',
     'pyls-black>=0.4.6',
     'pyls-spyder>=0.3.2,<0.4.0',
-    'pyqt5<5.13',
-    'pyqtwebengine<5.13',
     'python-language-server[all]>=0.36.2,<1.0.0',
     'pyxdg>=0.26;platform_system=="Linux"',
     'pyzmq>=17',
